client/my-sites/plugins/plugin-install-button-wpcom/index.jsx

```diff
@@ -56,9 +57,11 @@ export const WpcomPluginInstallButton = ( props ) => {
 		}
 	}
 
+	const label = isTransferring ? translate( 'Installing…' ) : translate( 'Install' );
```

Comment:
This looks great, and may also fix https://github.com/Automattic/wp-calypso/issues/52539

<img width="724" alt="Screen Shot 2021-05-17 at 2 46 46 PM" src="https://user-images.githubusercontent.com/1689238/118540595-bba53380-b71e-11eb-8f70-92c34fc99f49.png">

Reply:
Thanks. Issues linked.